src/Microsoft.AspNet.Mvc.ModelBinding/Metadata/DefaultBindingMetadataProvider.cs

```diff
@@ -53,6 +54,21 @@ public void GetBindingMetadata([NotNull] BindingMetadataProviderContext context)
                 context.BindingMetadata.PropertyBindingPredicateProvider = new CompositePredicateProvider(
                     predicateProviders);
             }
+
+            if (context.Key.MetadataKind == ModelMetadataKind.Property)
+            {
+                // BindingBehavior can fall back to attributes on the Container Type, but we should ignore
+                // attributes on the Property Type.
+                var bindingBehavior =
+                    context.PropertyAttributes.OfType<BindingBehaviorAttribute>().FirstOrDefault() ??
+                    context.Key.ContainerType.GetTypeInfo().GetCustomAttribute<BindingBehaviorAttribute>(inherit: true);
```

Comment:
Isn't this an additional behavior we are defining?
Can we do it separately? 
That will help us consider the alternatives like
Shouldn't the attribute on the type containing the property apply to whether a model of that type gets model bound or not?